src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
@weshaggard I removed this based on @ericstj 's suggestion. See https://github.com/dotnet/wcf/pull/793/files#r52515944

Is this a case of going with the lesser of two evils? What do you advise?